Grid/grid.css

```diff
@@ -9,43 +9,60 @@
 }
 
 .grid-wrapper {
-    margin: 4rem 2rem;
+    margin: 1rem;
+    padding: 1rem;
```

Comment:
If you've used utility classes for margin and padding then these properties can be directly added in your html tag. If not then it's fine. Just a suggestion.